securityscanutils/securityscan.go

```diff
@@ -248,8 +266,14 @@ func (r *SecurityScanRepo) RunMarkdownScan(ctx context.Context, release *github.
 		}
 
 	}
+	localOutputFilename := ""
+	if r.Opts.OutputResultLocally {
+		githubIssueOutputDir := path.Join(r.Opts.OutputDir, r.Repo, "github_issue_results")
+		err = os.MkdirAll(githubIssueOutputDir, os.ModePerm)
+		localOutputFilename = path.Join(githubIssueOutputDir, version+".md")
+	}
 	// Create / Update Github issue for the repo if a vulnerability is found
-	return r.githubIssueWriter.CreateUpdateVulnerabilityIssue(ctx, release, vulnerabilityMd)
+	return r.githubIssueWriter.CreateUpdateVulnerabilityIssue(ctx, release, vulnerabilityMd, r.Opts.AdditionalContext, localOutputFilename)
```

Comment:
Should this always be run? Now that the "vulnerablityAction" has 3 options:  "output-locally", "github-issue-all", "github-issue-latest", shouldn't we _only_ use the Github writer if `github-issue-all` or `github-issue-latest` is set?